src/api.js

```diff
@@ -1,4 +1,5 @@
 import rp from 'request-promise';
+import request from 'request';
```

Comment:
the download method I added streams the download to the file. `request-promise` recommends not using it for streaming but rather using plain `request`.